polar2grid/core/rescale_configs/rescale.ini

```diff
@@ -180,6 +180,13 @@ method=linear
 min_in=0.0
 max_in=50.0
 
+[rescale:default_sfr]
+data_kind=sfr
+method=linear
+min=0.0
+max_in=50.0
+#This should probibly be from 0 to 5, or some other max value, pending a conversation with the mirs team.
+
```

Comment:
@kathys Do you think you could contact the MIRS team about this and see if snow fall rate is really only useful between 0 and 5 mm/hr?

Reply:
@kathys I forget, did we ever get a response on this?

Reply:
On September 28, 2018, Jim Davies sent this to me from Chris Grassotti:
We were contacted directly by the Alaska folks about the SFR and gave them some guidance. The max value in the SFR (after applying the scale factor of 100 in the netcdf) is about 5 mm/h (stored as 500 in the file). The maximum possible value of 1000 in the metadata was a sort of placeholder and will probably be revised on a future release of MiRS.